BUILD

```diff
@@ -179,6 +179,7 @@ PYTHON3 |= {
 NODEJS_VERSIONS = [
     "20",
     "22",
+    "24",
```

Comment:
I thin you might still need to handle this section: https://github.com/GoogleContainerTools/distroless/blob/main/BUILD#L191-L219 for architectures

Reply:
Added it as well